stagecoach/price-pusher/price_pusher/configs/cli.py

```diff
@@ -0,0 +1,101 @@
+import click
+import os
+import logging
+
+from logging import Logger
+from typing import Optional, Union
+from pragma.publisher.client import (
+    PragmaClient,
+    PragmaOnChainClient,
+    PragmaAPIClient,
+)
+
+from price_pusher.utils.aws import fetch_aws_private_key
+
+
+def create_client(
+    target: str,
```

Comment:
this is probably not needed as we got CLI checks from `click` that returns an `str` anyway (we can't make it accept enums, checked that) & they're only used here right after.
however I created a type alias to restrict their content & make it more understandable which values it can hold:

```python
Target = Literal["onchain", "offchain"]
Network = Literal["mainnet", "sepolia"]
```